src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/FxmlController.java

```diff
@@ -42,7 +52,19 @@ private void initialize() {
     }
 
     @FXML
-    private void printOutput() {
-        outputText.setText(inputText.getText());
+    private void createJson() throws IOException {
```

Comment:
This method should not be part of the GUI class. All the controller should do is parsing the values from the input fields and then delegate those values to another object.

The other class then can instrument `BloxbergClient` and `TestCenter` to perform the task. This way the logic is decupled from actual GUI implementation and can be reused, e.g. for the CLI.